### DISCUSSION TITLE
Gdk or Gtk

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [GtkSharp/GtkSharp#428] [Q&A] [answered]
The nuget package and the project website, are each titled differently. 

GDK = Game Development Kit

https://learn.microsoft.com/en-us/gaming/gdk/

**Comment by badcel** [comment, marked as answer]
GTK is not just one library but a full stack of libraries one of it being GTK and another one GDK which is not related to the Microsoft one.

See the family of packages on nuget: https://www.nuget.org/profiles/GtkSharp

Or the GTK API documentation: https://docs.gtk.org

**Comment by lytico** [comment]
https://en.wikipedia.org/wiki/GTK

https://en.wikipedia.org/wiki/GDK
